<?xml version="1.0"?>
<launch>
  <arg name="side" default="r"/> <!-- "b" = blue side, "r" = red side  -->
  <arg name="judge_url" default="http://localhost:5000/"/>
    <!-- You can change control program  -->

    <!-- sample program node -->
    <param name="judge_url" value="$(arg judge_url)"/>
    <param name="cf_side" value="$(arg side)"/>
    <node pkg="burger_war_dev" type="randomRun.py" name="randomRun" output="screen"/>
    <node pkg="burger_war_dev" type="judge_listener.py" name="judge_node" output="screen"/>
    
    <!-- END sample program node -->

    <!-- Navigation sample program  -->
    <!--
    <include file="$(find burger_navigation)/launch/burger_navigation.launch" />
    <node pkg="burger_war_dev" type="navirun.py" name="navirun" output="screen"/>
    -->
    <!-- END Navigation sample program node -->

    <!-- End of your space  -->
</launch>

